### PR TITLE
CIV-3664 Refactor Judge Makes Decision to Make Decision for reusability

### PR DESCRIPTION
**Pull request by karthick-mohanasundaram-HMCTS**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CIV-3664

### Change description ###

Refactor Judge Makes Decision to Make Decision for reusability (Caseworker and judge)

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -7,7 +7,7 @@ import uk.gov.hmcts.contino.AppPipelineDsl
 def type = "java"
 def product = "civil"
 def component = "general-apps-ccd-definition"
-def camundaBranch = "master"
+def camundaBranch = "feature/CIV-3664"
```
Comment (karthick-mohanasundaram-HMCTS) [inline Jenkinsfile_CNP:10]: - [ ] Revert

**Comment by karthick-mohanasundaram-HMCTS** [comment]
image

**Comment by karthick-mohanasundaram-HMCTS** [comment]
image

**Comment by karthick-mohanasundaram-HMCTS** [comment]
Test are failing due to unable to find the enum. New enum is available in bankend which is raised in the pr - https://github.com/hmcts/civil-general-applications/pull/466. it should pass once it's backend is merged master

**Review by vasudevganesanhmcts — APPROVED** [review]

**Pull request merged**